_python_workout_code/flexibleDict.py

```diff
@@ -0,0 +1,12 @@
+class FlexibleDict():
+    def __init__(self):
+        self.dict = {}
+
+    def __setitem__(self, key, value):
+        self.dict[str(key)] = value
+
+    def __getitem__(self, key):
+        try:
+            return self.dict[str(key)]
```

Comment:
speaking of a `dict` data type, in python it has a `.get(key)` method, that will return `None` by default, if the key wasn't found in the dict instance: https://docs.python.org/3/library/stdtypes.html#dict.get